src/runtime/vulkan/vulkan_stream.h

```diff
@@ -43,135 +44,64 @@ struct VulkanStreamToken {
   std::vector<VkBuffer> buffers_;
 };
 
+/*!
+ *  \brief Wrapper around a vulkan command buffer
+ *
+ *  The VulkanStream collects commands into a VkCommandBuffer.  When a
+ *  newly submitted command requires resources reserved by an
+ *  already-submitted command, all of the queued commands are
+ *  submitted to the GPU, and the CPU waits for all queued commands to
+ *  finish.  The queued commands can also be explicitly pushed/waited
+ *  on by calling VulkanStream::Synchronize.
+ *
+ *  If the device is
```

Comment:
Nice work with the additional comments here, but it seems this paragraph is incomplete!

Reply:
Thank you for that catch, and next time I should probably separate out the documentation from the refactoring to make those items be easier to find ahead of time.  I have no memory of what I was going to finish that paragraph with, so I have deleted it for now.

Also, fixed up the linting errors.